examples/universal-with-router/client/index.js

```diff
@@ -0,0 +1,25 @@
+import 'babel-polyfill'
+import React from 'react'
+import { render } from 'react-dom'
+import { Provider } from 'react-redux'
+import { Router, browserHistory } from 'react-router'
+import { syncHistoryWithStore } from 'react-router-redux'
```

Comment:
From my earlier comment:

>Rename universal to async-universal. Again, **no need for react-router-redux there.**

You can remove it :wink: . I already added it to the only example where it makes sense: `real-world`.